### PR TITLE
Fix: #172, Enhancement, Create aria a11y question and answer 

### PR DESCRIPTION
**Pull request by digilou**
Created an accessibility question with answer about WAI-ARIA.

## Description

**Adds to** #172  

## What does your PR belong to?

* [x] Questions / Answers
* [ ] Website
* [ ] General / Things regarding the repository (like CI Integration)

## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [x] Enhancement (non-breaking improvement of a question)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

* [x] My code follows the code style of this project.
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [x] I have checked that the changes are working properly
* [x] I have checked that there isn't any PR doing the same
* [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: questions/accessibility-aria.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+An example using ARIA:
+
+```
+<div 
+  role="combobox"
+  aria-expanded="false"
+  aria-owns="ex1-grid"
+  aria-haspopup="grid"
+  id="ex1-combobox">
+  ...
```
Comment (fejes713) [inline questions/accessibility-aria.md]: You can remove these `...` I guess. 😃

File: questions/accessibility-aria.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+* Provides role, name, and state
+* Semantic HTML that doesn't produce interaction by itself
+
+##### Additional links
+
+* [WAI-ARIA Overview](https://www.w3.org/WAI/standards-guidelines/aria/)
+* [WAI-ARIA Spec](https://www.w3.org/TR/wai-aria/)
+* [ARIA Serious? Eric Eggert presentation](https://youtu.be/4bH57rWPnYo)
+
+tags: accessibility, web applications, aria
```
Comment (fejes713) [inline questions/accessibility-aria.md]: Please follow the guidelines regarding the `tags`. Currently, we will only support `accessibility`, and not others that you included. 

Other than that everything seems in place 👍 Thank you so much!

File: questions/accessibility-aria.md (hunk 1 above)
Comment (digilou) [reply questions/accessibility-aria.md]: No problem. An indicator that more code should go there, but prob not the best way to express that idea.

**Comment by digilou** [comment]
Suggested updates made @fejes713

File: questions/accessibility-aria.md (hunk 2 above)
Comment (digilou) [reply questions/accessibility-aria.md]: Ah. Sorry about that. I didn't read closely, and was thinking about future tags.

File: questions/accessibility-aria.md (hunk 1 above)
Comment (fejes713) [reply questions/accessibility-aria.md]: Oh my wrong .... Sorry, I'll fix this.

**Comment by fejes713** [comment]
Thank you so much @digilou :octocat: This category is coming along nicely! I'll see to make it live by the end of the next week 👍 

Feel free to add more questions if you want to 😄 Have a nice day!

**Comment by fejes713** [comment]
Ah, 😕 I accidentally closed it. Can you do a PR once again 😭 Sorry

**Comment by atomiks** [comment]
Is it just me or is this PR glitched?

It says "Merged" on the PR list, "Open" at the top, and "Closed" at the bottom 😆

**Comment by fejes713** [comment]
Same happened to me 😕 Haha, it could be but I remember I pressed merge and it acted like close 😭 

@atomiks btw. feel free to take a quick look at GitHub Universe talk repo ❤️

**Comment by digilou** [comment]
@fejes713 LOL I'll submit a new PR tonight, along with another a11y question I've been working on.

File: questions/accessibility-aria.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+### What is ARIA and when should you use it?
+
+#### Answer
+
+ARIA stands for "Accessible Rich Internet Applications". ARIA code is HTML. It provides additional attributes in the development of web applications to offer screen reader users more context about dynamic components, including the component's role, name, and state. It should only be used when an HTML element equivalent is not available or lacks full support. ARIA is semantic, and doesn't actually make your component dynamic.
```
Comment (skatcat31) [inline questions/accessibility-aria.md]: ARIA code is HTML and JavaScript isn't it? There's many things you can do with ARIA you can't do without JavaScript

File: questions/accessibility-aria.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  id="ex1-combobox">
+  ...
+</div>
+```
+Credit: W3C's [ARIA 1.1 Combobox with Grid Popup Example](https://w3c.github.io/aria-practices/examples/combobox/aria1.1pattern/grid-combo.html)
+
+#### Good to hear
+
+* Accessible Rich Internet Applications
+* Benefits screen reader users
```
Comment (skatcat31) [inline questions/accessibility-aria.md]: Benefits users with accessibility needs*

This includes:
- Motor
- Cognitive
- Visibility
- Hearing
- Alternative inputs

File: questions/accessibility-aria.md (hunk 3 above)
Comment (digilou) [reply questions/accessibility-aria.md]: You're correct that ARIA is mostly useless without JavaScript, but ARIA itself is HTML attributes.

File: questions/accessibility-aria.md (hunk 4 above)
Comment (digilou) [reply questions/accessibility-aria.md]: That's why I stated screen reader "users", which can include more audience than just visually impaired. If it helps devs see a clearer picture, I'm all for clarity.

Comment (digilou) [reply questions/accessibility-aria.md]: Maybe...?
"Benefits screen readers users, such as people with visual and cognitive impairments"

Comment (skatcat31) [reply questions/accessibility-aria.md]: I would say `Benefits users with assistive technology needs` at the very least.

This includes switch users, screen-readers, and touch interface, and mobility, and cognitive.

File: questions/accessibility-aria.md (hunk 3 above)
Comment (digilou) [reply questions/accessibility-aria.md]: I was torn about "WAI-ARIA" too. I think you're right that WAI-ARIA is the most correct way to teach people about it. I was thinking from the point of view of devs who hear it called ARIA and could feel more familiar with the question.

File: questions/accessibility-aria.md (hunk 4 above)
Comment (digilou) [reply questions/accessibility-aria.md]: I've always been taught it was created for the benefit of screen reader users, but the spec does say assistive technology, so let's go with that.

File: questions/accessibility-aria.md (hunk 3 above)
Comment (digilou) [reply questions/accessibility-aria.md]: I should probably think of it as a standard, too. The attributes are a way to create accessible web apps. The spec specifically states:
> WAI-ARIA is a technical specification that provides a framework to improve the accessibility and interoperability of web content and applications.

Comment (digilou) [reply questions/accessibility-aria.md]: This is why I think of it as code:
> The incorporation of WAI-ARIA is a way for an author to provide proper semantics for custom widgets to make these widgets accessible, usable, and interoperable with assistive technologies.

**Comment by digilou** [comment]
Oh weird. It says I closed this. Must've been when I was updating my branch.

**Pull request merged**